hacking/build_library/build_ansible/command_plugins/plugin_formatter.py

```diff
@@ -333,6 +334,17 @@ def jinja2_environment(template_dir, typ, plugin_type):
     return templates
 
 
+def process_version_added(version_added):
+    if not isinstance(version_added, str):
```

Comment:
```suggestion
    if not isinstance(version_added, string_types):
```

Reply:
Darn, looks like I forgot about it :) Thanks for spotting this!